lib/patch-series.ts

```diff
@@ -68,7 +70,9 @@ export class PatchSeries {
                 }
             });
 
-            rangeDiff = await git(["range-diff", "--no-color", range]);
+            if (await gitCommandExists("range-diff", workDir)) {
```

Comment:
What happens if `rangeDiff` stays uninitialized?

Reply:
(I see higher up that it is initialized to "", so things probably aren't _broken_, but not great)

Reply:
If it is empty, or false, or undefined, nothing gets inserted in `insertRangeDiff()`: https://github.com/dscho/gitgitgadget/blob/2fb305a285398b8ac957ef35f51e0be5e295c63a/lib/patch-series.ts#L356-L358